ui/app/AppLayouts/Onboarding2/KeycardCreateProfileFlow.qml

```diff
@@ -113,17 +113,13 @@ SQUtils.QObject {
     Component {
         id: backupSeedRevealPage
         BackupSeedphraseReveal {
-            Component.onCompleted: {
-                try {
-                    d.mnemonic = root.generateMnemonic()
-                    root.seedphraseSubmitted(d.mnemonic)
-                } catch (e) {
-                    console.error('failed to generate mnemonic', e)
-                }
-            }
+            Component.onCompleted: d.mnemonic = root.generateMnemonic()
             mnemonic: d.mnemonic
 
-            onBackupSeedphraseConfirmed: root.stackView.push(backupSeedVerifyPage)
+            onBackupSeedphraseConfirmed: {
+                root.seedphraseSubmitted(d.mnemonic)
```

Comment:
we report as submitted sth that we received from outside. Is it necessary?

Reply:
We collect it... but not sure what's the prereq from the backend side, @igor-sirotin ?